src/main/resources/errors.yaml

```diff
@@ -871,6 +871,17 @@ request-errors:
       
       Resend the command using only columns defined in the table schema.
 
+  - scope: SCHEMA
+    code: UNKNOWN_INDEX_TYPES
+    title: Index type is unknown
+    body: |-
+      The command attempted to create an index using an unknown type.
+      
+      The supported index types are: ${supportedTypes}.
```

Comment:
I think these are all known types but not all of them are necessarily supported. So maybe should be:

    The known index types are: ${knownTypes}